web/experience.html

```diff
@@ -128,6 +128,17 @@ <h6 id="notesTitle" class="fieldNotesInnards is-hidden" ><span class="theUserNam
                     
                     <label>Provide a description of what prompted you to do this</label>
                     <textarea deer-key="description"></textarea>
+
+                    <div class="card bg-light">
+                        <header class="text-primary">Linked Media</header>
+                        <p class="text-primary"><small>Provide the URL of representations of this event. 

```

Comment:
Maybe change the wording here.  `Provide a URL for this event, if applicable.  Media specific to an object or person should not be added here.`

Reply:
hmm... I'm not sure here. Your suggestion is confusing because we want the URL for the media that is being linked.

Reply:
Haha.  ok ok, make it very clear that what I really want is a cat pic.  I see what you're saying.  This is not "let me have the URL of the website that was made for this event".  It is more of a "here is ambient crowd noise from this event".

Reply:
Suggesting `Optionally link to media from this event.` and remove the "link" restatement from the following.

Reply:
So maybe like `Provide the URL(S) for media representing this event`.   `URL representations of this event` just doesn't feel right, it is not clear what is being asked for.

Reply:
Right now, we haven't put in array handling, so it is `URL` not `URL(s)`. The suggestion I made is not "URL representations" the code is marked "outdated" above this.

Reply:
I also want it to be clear that we are _not_ asking for images of the objects, recordings of the songs, photos on the site, etc. but specifically images of the Experience. I'm sure that will not be done correctly anyway, but I'd like them to think it was their fault.

Reply:
Sorry I hadn't refreshed!  O.K.  `Provide the url of representations of this event`.  Is that clear enough to differentiate https://schema.org/image from https://schema.org/url?  For me, I still don't know which you are asking for.  URL to some media file that represents this event, or the URL for the event.